applications/launchpad_v2/src/styles/themes/index.ts

```diff
@@ -1,9 +1,16 @@
 import darkTheme from './dark'
 import lightTheme from './light'
 
+const withShared = theme => ({
+  ...theme,
+  borderRadius: (count = 1) => `${count * 10}px`,
```

Comment:
Cool! Just curious: how did you pick up the base values (`10px`, `0.7px` and `1.3px`)?

Reply:
pure guess
looked good, but I thought this way it will be easier to adjust them later ^^